### PR TITLE
Minor barrier amendments

### PR DESCRIPTION
**Pull request by matthew-andrews**
screen shot 2015-07-30 at 17 22 00

### CODE REVIEW

File: views/article-v2.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
 		<div class="ad-placeholder__leaderboard ad-placeholder__{{__name}}"></div>
 		<div class="article__header-inner o-grid-row">
 			{{#if barrier}}
-			<div data-o-grid-colspan="12 L12 XL12">
+			<div data-o-grid-colspan="12 L10 XL12">
```
Comment (matthew-andrews) [inline views/article-v2.html]: oops didn't mean to do this

**Comment by matthew-andrews** [comment]
screen shot 2015-07-30 at 17 36 22

**Comment by keirog** [comment]
:+1: Happy to see `nextLinkButton` go

**Comment by matthew-andrews** [comment]
still there for some bits… baby steps

**Pull request merged**